website/content/api-docs/jobs.mdx

```diff
@@ -1246,6 +1246,8 @@ The table below shows this endpoint's support for
 enabled, this value must match a namespace that the token is allowed to
 access. This is specified as a query string parameter.
 
+- `reverse` `(bool: false)` - Specifies the list of returned allocations to be sorted in the reverse lexicographical order. By default, they are lexicographically sorted.
```

Comment:
```suggestion
Added "by ID", and also we hard-wrap markdown text:

```suggestion
- `reverse` `(bool: false)` - Specifies the list of returned allocations to be
  sorted in the reverse lexicographical order. By default, they are
  lexicographically sorted by ID.
```
```